src/main/java/org/jitsi/videobridge/SctpConnection.java

```diff
@@ -978,9 +978,10 @@ public void removeChannelListener(WebRtcDataStreamListener listener)
     private void runOnDtlsTransport(StreamConnector connector)
         throws IOException
     {
-        DtlsControlImpl dtlsControl
-            = getTransportManager().getDtlsControl(this);
-        DtlsTransformEngine engine = dtlsControl.getTransformEngine();
+        SrtpControl dtlsControl
```

Comment:
Well, it really is a DtlsControl (otherwise the cast below wouldn't work). I think the name of the variable could be a useful clue.